usaspending_api/etl/es_award_template.json

```diff
@@ -1,22 +1,20 @@
 {
-  "index_patterns": null,
   "settings": {
     "index.mapping.ignore_malformed": true,
     "index.max_result_window": null,
     "index.refresh_interval": -1,
     "index": {
       "number_of_shards": 5,
-      "number_of_replicas": 0
+      "number_of_replicas": 0,
+      "sort.field" : ["total_obligation.keyword", "action_date"],
```

Comment:
Need to change to `award_id` instead of `action_date` or merge in the PR from #2273

Reply:
Ok starting on now

Reply:
Doing it now thanks alot